tests/accessor/host_accessor_api_common.h

```diff
@@ -116,13 +116,13 @@ class run_api_tests {
       SECTION(get_section_name<dims>(
           type_name, access_mode_name,
           "Check api for ranged host_accessor with offset")) {
-        constexpr size_t acc_range_size = 4;
-        constexpr size_t buff_range_size = 8;
-        constexpr size_t buff_size = (dims == 3)   ? 8 * 8 * 8
-                                     : (dims == 2) ? 8 * 8
-                                                   : 8;
-        constexpr size_t offset = 4;
-        constexpr size_t index = 2;
+        constexpr size_t acc_range_size = 2;
+        constexpr size_t buff_range_size = 4;
+        constexpr size_t buff_size = (dims == 3)   ? 4 * 4 * 4
```

Comment:
Add a comment from your PR description explaining the constraints, just in case some day someone find these numbers too small and want to increase them again.

Reply:
Done https://github.com/KhronosGroup/SYCL-CTS/pull/781/commits/7ee72365af0bfa49a6b93535e410c599057d1f08